tools/project-context.js

```diff
@@ -1580,3 +1598,100 @@ _.extend(exports.FinishedUpgraders.prototype, {
     files.appendFile(self.filename, appendText);
   }
 });
+
+export class MeteorConfig {
+  constructor({
+    appDirectory,
+  }) {
+    this.appDirectory = appDirectory;
+    this.packageJsonPath = files.pathJoin(appDirectory, "package.json");
+    this.watchSet = new watch.WatchSet;
+    this._resolversByArch = Object.create(null);
+  }
+
+  _ensureInitialized() {
+    if (! _.has(this, "_config")) {
```

Comment:
This file certainly isn't void of Underscore usage, but any particular reason for introducing new `_` usage here?

Reply:
There's just so much `_` already there, I'm sure someone will clean this up whenever `project-context.js` gets modernized.